new/LIT6705CompGM.xml

```diff
@@ -0,0 +1,80 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6705CompGM" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">The Synaxarion-type compilation of ʾAbuna Gabra Māryām of Dabra Bizan</title>
+                <title xml:lang="en" xml:id="t2">Sǝnkǝssār of Dabra Bizan</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DR"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>A compilatory work of hagiographical notices and biblical excerpts, arranged in commemorations according to the days of the year as the synaxarion. It contains
```

Comment:
The wording "a compilatory work ... as the synaxarion" is a little difficult to grasp, it would be clearer if you say that this is "a synaxarion made of hagiographical notices and biblical excerpts"..."according to the church calendar" (even though a synaxarion is always arranged according to the church calendar, so it is in fact redundant). Is it what is meant?

Reply:
It is a complex situation. I want to be clear that this work is very far removed from the synaxarion as transmitted in the Ethiopic tradition (though based upon it and sharing the same structure) and constitutes a creative composition by Gabra Māryām.